src/components/Option/readme.md

```diff
@@ -69,3 +69,39 @@ initialState = { value: null };
     </GlobalHeader>
 </div>
 ```
+
+##### Option accept number on name prop
+
+```js
+import React from 'react';
+import { Picklist, Option } from 'react-rainbow-components';
+
+const containerStyles = {
+    width: '200px',
+};
+
+initialState = { value: { name: 1, label: 'Experimental Building' } };
+
+<div className="rainbow-m-bottom_xx-large rainbow-p-bottom_xx-large">
+    <GlobalHeader
+        src="images/user/user2.jpg"
+        className="rainbow-p-bottom_xx-large rainbow-m-bottom_xx-large"
+    >
+        <div className="rainbow-flex rainbow-align_right">
+            <Picklist
```

Comment:
@yvmunayev I think we should modify one of the existing examples instead add one new